### ISSUE TITLE
*Warning*: compute.thermo already specified for a group with name __nvt_all

### ISSUE THREAD

**Issue by drozzy** [glotzerlab/hoomd-blue#515]
I keep getting this warning:

```
*Warning*: compute.thermo already specified for a group with name __nvt_all
```

whenever I do:

```
integrator = hoomd.md.integrate.nvt(group=all, kT=0.1, tau=tau)
integrator.set_params(kT=target_kT)
```

as I set this pretty often, the warnings flood my console completely.

**Comment by joaander** [comment]
This will be solved with the v3.0 API redesign.

As a workaround, call `set_params` when you need to change the parameters of the integrator without instantiating a new integrator.

**Comment by drozzy** [comment]
Yeah, I call `set_params` repeatedly now but when I need to re-initialize my integrator, I do this (I guess this is when warnings come from?):

```
integrator.disable()
integrator = hoomd.md.integrate.nvt(group=all, kT=0.1, tau=tau)
```

**Comment by joaander** [comment]
PR #646 fixes this for v3.0.

**Comment by joaander** [comment]
When #696 is implemented, ComputeThermo will have a full v3.0 python interface that is not tied to the integrators.
